instrumentation/rocketmq/rocketmq-client/rocketmq-client-4.8/library/src/main/java/io/opentelemetry/instrumentation/rocketmqclient/v4_8/TracingConsumeMessageHookImpl.java

```diff
@@ -30,12 +34,9 @@ public void consumeMessageBefore(ConsumeMessageContext context) {
     Context parentContext = Context.current();
     Context newContext = instrumenter.start(parentContext, context.getMsgList());
 
-    // it's safe to store the scope in the rocketMq trace context, both before() and after() methods
-    // are always called from the same thread; see:
-    // - ConsumeMessageConcurrentlyService$ConsumeRequest#run()
-    // - ConsumeMessageOrderlyService$ConsumeRequest#run()
```

Comment:
I think this comment is still (mostly) relevant (and useful since its dangerous to store the active scope somewhere)

Reply:
restored comment